menas/src/main/scala/za/co/absa/enceladus/menas/controllers/DatasetController.scala

```diff
@@ -44,6 +46,12 @@ class DatasetController @Autowired()(datasetService: DatasetService)
 
   import scala.concurrent.ExecutionContext.Implicits.global
 
+  @GetMapping(Array("/latest"))
+  @ResponseStatus(HttpStatus.OK)
+  def getAll(@RequestParam(value = "missing_property", required = false) missingProperty: Optional[String]): CompletableFuture[Seq[Dataset]] = {
```

Comment:
This should not be a full Dataset. We do not have any pagination and on prod environments, this will be too huge. Could it be just Dataset name + version? Nothing else really matters.